driver/razerkbd_driver.c

```diff
@@ -365,6 +366,9 @@ static void razer_set_device_mode(struct usb_device *usb_dev, unsigned char mode
     case USB_DEVICE_ID_RAZER_ORNATA:
     case USB_DEVICE_ID_RAZER_ORNATA_CHROMA:
     case USB_DEVICE_ID_RAZER_ORNATA_V2:
+    case USB_DEVICE_ID_RAZER_ORNATA_V3_X:
+        request.transaction_id.id = 0x1F;
+        break;
```

Comment:
please don't modify behavior of existing devices. And anyways a 0x1F case is found a few lines below